frontend/src/modules/tournaments-list/pages/TournamentsListPage,js.js

```diff
@@ -0,0 +1,16 @@
+import ContentBox from '../../../shared/design-system/atoms/ContentBox';
```

Comment:
ty tečky můžeš všude vyměnit za src/

Reply:
jo, to bude lepší kdyby se ty soubory přemysťovali. To za mě geenruje IntelliJ tak jsem na to ani nekoukal :D